src/scripts/main.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  const onLeftClick = (event) => {
+    if (event.button === 0) {
+      resolve('First promise was resolved');
+    }
+  };
+
+  document.addEventListener('mousedown', onLeftClick);
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
```

Comment:
The first promise will be rejected after 3 seconds regardless of whether the left mouse button is clicked. Consider clearing the timeout if the promise is resolved to prevent it from being rejected after resolution.